handbook/engineering/organization.md

```diff
@@ -0,0 +1,35 @@
+# Organization
+
+The purpose of our engineering organization is to build a product that is valuable to our customers.
+
+To achieve this purpose, we need to ensure two things:
+
+1. Each feature or product that we build needs to coherently solve a particular customer problem. {#customer-needs}
```

Comment:
is `{#customer-needs}` supposed to link somewhere? Same with `{#engineering-needs}` in the next line.

Reply:
The opposite: this creates an anchor on the page so that I can link to this part of the document from elsewhere.

Reply:
🤯 TIL